### DISCUSSION TITLE
Decision to not Use Scala in Implementation

### DISCUSSION THREAD

**Discussion by MtgSaber** [fnuecke/oc2#112] [General]
Hey, just curious about the reasoning behind not using Scala in the implementation, since OC1 uses it for functionality.

**Comment by fnuecke** [comment]
Couple of things. It was a massive blocker for contributors. There's no built-in support in Forge anymore (I know it's possible, but makes things more complicated). I just felt like going back to good old plain Java for a bit.

**Reply by MtgSaber** [reply]
Ah, makes sense. Scala certainly isn't very widespread yet. I only really got into it after seeing it in OC1 and hearing a few colleagues praising it. I do wish it would be more popular though; it's a very fun language. Thanks for sating my curiosity!
